examples/gerry_learning_auditing.py

```diff
@@ -0,0 +1,46 @@
+# testing integration of gerry fair with aif360
```

Comment:
Can this be turned into a full-on test script that can be run by pytest and placed in the `tests/` directory?